forum/forum.py

```diff
@@ -13,345 +13,389 @@
 from flask_login.login_manager import LoginManager
 from werkzeug.security import generate_password_hash, check_password_hash
 
+from flask_images import *
+
+images = Images(app)
 
 db = SQLAlchemy(app)
 
-#VIEWS
+
+# VIEWS
 @app.route('/profile')
 def profile():
-	return render_template("profile.html")
+    user_id = int(request.args.get("user"))
+    user = User.query.filter(User.id == user_id).first()
+    if not user:
+        return render_template("error.html", error="That User Does Note Exist")
+
+    return render_template("profile.html", user=user)
+
 
```

Comment:
Sort of working profile viewer, no default, simple error page if user doesn't exist.